core-api/src/main/java/com/optimizely/ab/config/ProjectConfig.java

```diff
@@ -544,13 +543,13 @@ public boolean setForcedVariation(@Nonnull String experimentKey,
      * Gets the forced variation for a given user and experiment.
      *
      * @param experimentKey The key for the experiment.
-     * @param userId The user ID to be used for bucketing.
-     *
+     * @param userId        The user ID to be used for bucketing.
      * @return The variation the user was bucketed into. This value can be null if the
      * forced variation fails.
      */
-    public @Nullable Variation getForcedVariation(@Nonnull String experimentKey,
-                                                  @Nonnull String userId) {
+    public @Nullable
+    Variation getForcedVariation(@Nonnull String experimentKey,
```

Comment:
Same as other comments about annotation placement for return types. This is formatting is worse.